pyproject.toml

```diff
@@ -32,6 +32,7 @@ dependencies = [
   "PyGithub",
   "GitPython",
   "psm-utils",
+  "python-calamine"
```

Comment:
They seem to also provide further dependencies for pandas directly: https://pandas.pydata.org/docs/getting_started/install.html#performance-dependencies-recommended

But if it works, all good